CMakeLists.txt

```diff
@@ -1,44 +1,89 @@
-cmake_minimum_required(VERSION 3.4...3.22)
-project(BTLS VERSION 1.3.7)
+cmake_minimum_required(VERSION 3.15...3.26)
 
-# set macro
-if(WIN32)
-  option(Win "|To build on Windows|" ON)
+# Set macros
+option(BTLS "Build as a C++ terminal program" ON)
+option(PyBTLS "Build as a Python package" OFF)
+option(WIN_DEBUG "Debug BTLS on Windows" OFF)
+option(DEBUG_BUILD "Enable PyBTLS debug build" OFF)
+if(PyBTLS)
+  set(BTLS OFF)
 endif()
 
+# Set project name and version
```

Comment:
Fantastic - have you tested build as a binary (non-python) locally? Does it work on just windows or more?

Reply:
Works as usual.